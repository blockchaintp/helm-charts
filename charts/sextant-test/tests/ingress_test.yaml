suite: sextant-test.test the ingresses
templates:
  - charts/sextant/templates/ingress.yaml
release:
  name: test
  namespace: testns
tests:
  - it: ingress links to service properly
    set:
      sextant:
        ingress:
          enabled: true
          hosts:
            - host: test-host
              paths:
                - /path
          apiVersion: extensions/v1beta1
    asserts:
      - equal:
          path: spec.rules[0].http.paths[0].backend.serviceName
          value: test-sextant-test
      - equal:
          path: spec.rules[0].http.paths[0].backend.servicePort
          value: 8000
      - matchSnapshot:
          path: spec
