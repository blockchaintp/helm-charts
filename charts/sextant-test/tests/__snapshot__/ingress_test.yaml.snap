ingress links to service properly:
  1: |
    rules:
    - host: test-host
      http:
        paths:
        - backend:
            serviceName: test-sextant-test
            servicePort: 8000
          path: /path
